doc/en/usage.rst

```diff
@@ -174,7 +174,11 @@ This will add an extra property ``example_key="1"`` to the generated
 
 .. code-block:: xml
 
-    <testcase classname="test_function" example_key="1" file="test_function.py" line="0" name="test_function" time="0.0009">
+    <testcase classname="test_function" file="test_function.py" line="0" name="test_function" time="0.0009">
+      <properties>
```

Comment:
I think that custom_properties is good idea, and I am happy you implemented.
We use it to store various info about testcase which can be post-processed after test execution.
Like Bugzilla/Jira ID for skipped tests, Polarion ID, parameters of generated tests ...

I like this way (<properties>) more than attributes of <testcase>, but it is my personal opinion.
If you consider to include my patch, I will put back the warning to doc, and change commit message.